src/tests/test_integration/test_extra_functionality_movie.py

```
@@ -0,0 +1,241 @@
+import pytest
+from httpx import AsyncClient
+
+from tests.conftest import db_session
```

Ruff (0.11.9)

`tests.conftest.db_session` imported but unused

Remove unused import: `tests.conftest.db_session`

(F401)